pkg/client/add_events.go

```diff
@@ -87,6 +96,26 @@ func (client *DataSetClient) AddEvents(bundles []*add_events.EventBundle) error
 	return nil
 }
 
+func (client *DataSetClient) fixServerHostsInBundles(bundles []*add_events.EventBundle) {
```

Comment:
Would be good to add a comment which explains what this function does and why it's needed.

Reply:
I have added the comment.